.github/workflows/cleanup-artifacts.yml

```diff
@@ -0,0 +1,19 @@
+name: Generated APK Clean
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    # Every Monday at 1am
+    - cron: '0 1 * * 1'
+
+jobs:
+  remove-old-artifacts:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Clean all artifacts
+        uses: c-hive/gha-remove-artifacts@v1
```

Comment:
Hmmm this action doesn't seem to be created by a verified user and offiical account. We would need to request a secruity check via freshservice before using it.

Reply:
I will remove the flow for now and then, once we verify better, reintroduce it